ec2/spark_ec2.py

```diff
@@ -330,7 +338,10 @@ def get_spark_ami(opts):
         print >> stderr,\
             "Don't recognize %s, assuming type is pvm" % opts.instance_type
 
-    ami_path = "%s/%s/%s" % (AMI_PREFIX, opts.region, instance_type)
+    # URL prefix from which to fetch AMI information
+    ami_prefix = "{r}/{b}/ami-list".format(r=opts.spark_ec2_git_repo.replace("github","raw.github"), b=opts.spark_ec2_branch)
```

Comment:
What if GitHub is using Spark and they have a copy under `github/spark-ec2`?

I'd say to be safe/defensive, match on `https://github.com` and only at the very start of the string.